README.md

```diff
@@ -282,6 +291,29 @@ The command to fine-tune the model with Intel® optimizations enabled is:
 ipexrun --use_logical_core --enable_tcmalloc src/finetune_model.py --model_config configs/config_base.yml --data_path data/abcnews-date-text.csv --save_path saved_models/gpt2-medium-finetuned-intel --intel
 ```
 
+**For intel xpu training**, 'xpu' device is needed to add to 'python/site-packages/transformers'.    
+It can be changed manually as the following:
+```python

```

Comment:
Please update the diff with the latest one (one that also has "import intel_extension_for_pytorch as ipex")